components/brave_new_tab_ui/components/privateTab/index.tsx

```diff
@@ -35,7 +48,7 @@ export default class NewPrivateTab extends React.PureComponent<Props, {}> {
   render () {
     const { isTor } = this.props
     return (
-      <Page isPrivate={!isTor}>
+      <Page isPrivate={!isTor && !this.isQwant}>
```

Comment:
It's not private if it's `de` or `fr` language?

Reply:
this is just to change the background color. I should have written `isDefaultPrivate` but need to level up my naming game :P

Reply:
@cezaraugusto that's ok, when things calm down we should look to improve some naming all around the webui's